enplot/run.py

```diff
@@ -53,6 +53,12 @@ def main():
                         help=("comma-separated list of column index in " +
                               "the data file for use as Y variables"),
                         type=str)
+    parser.add_argument("-logx",
+			help=("plot using log of the values in x-axis"),
+			action='store_true', default=False)
+    parser.add_argument("-logy",
```

Comment:
lets make these `--log-x` and `--log-y` to comply with standard unix argument name scheme for multicharacter arguments.